edgelet/build/charts/edge-kubernetes/values.yaml

```diff
@@ -74,13 +74,15 @@ edgeAgent:
     enableK8sServiceCallTracing: false
     # Configure edge agent log verbosity
     runtimeLogLevel: 'debug'
-    # Configure the persistent volume settings.
+    # Configure the default claim size to use when creating PVCs.
     # persistentVolumeClaimDefaultSizeInMb: ""
-    # Use either 'persistentVolumeName' or 'storageClassName'
-    # If you set both, 'persistentVolumeName' will be preferred.
-    # 'persistentVolumeName' will reference a PersistentVolume on the cluster
+    # Use either 'persistentVolumeName' or 'storageClassName' to use when
+    # creating PVCs.
+    # The storageClassName is allowed to be an empty string. Since Helm doesn't
+    # provide a clean way of specifying an empty string (see: https://github.com/helm/helm/issues/2600)
```

Comment:
That's mildly annoying.

Reply:
Should we make it clear:
to use a storage class, set storageClassName.
to use a PV, set persistentVolumeName and storageClassName to "-"

Reply:
Setting `storageClassName` to `"-"` is not mandatory when using PVs. It so happens what when using statically provisioned PVs in Azure AKS with Azure Files backed storage, the storage class needs to be set to an empty string in order for it to work (i.e. to prevent it from using the default storage class). I think the text here can be made clearer though. Will do that.